services/context/context_response.go

```diff
@@ -13,6 +13,7 @@ import (
 	"path"
 	"strconv"
 	"strings"
+	"syscall"
```

Comment:
Will this work on Windows? My impression is that the syscall package is unix only.

Reply:
It is a common package.

Windows also have similar concept "syscall"

![image](https://github.com/go-gitea/gitea/assets/2114189/2449f561-3982-4314-8faf-814a4119ae5a)

Reply:
Yeah looks like there is some form of stub module for windows syscall. I don't think it could emit EPIPE there but at least it will compile.